sem/database.py

```diff
@@ -206,6 +207,16 @@ def insert_results(self, results):
             'meta': {k: ['...'] for k in ['elapsed_time', 'id', 'exitcode']},
```

Comment:
There might be a problem if we do this. I replied regarding this on @mattia-lecci's suggestion above. 
Here:
> The `DatabaseManager.have_same_structure()` recursively looks for keys if there are nested dictionaries present inside the parent dictionary. If we just want to add `log_component` as a key, then we might also need to modify this function to prevent it from recursively looking inside the `log_component` dictionary.

Reply:
right, [these](https://github.com/signetlabdei/sem/blob/b704be8c82fee6411d5aba98d24079dcb58d1fd9/sem/database.py#L496-L498) are the incriminated lines actually. any thought on this @DvdMgr ? maybe adding a flag `recurseDictionaries`?

Reply:
Replied to this in the original comment above